packages/core/src/StoreManager.test.ts

```diff
@@ -138,50 +130,56 @@ describe('Cevitxe', () => {
 
       // confirm the store is gone
       expect(localStoreManager.store).toBeUndefined()
+      expect.assertions(2)
     })
 
     it('should persist state between sessions', async () => {
-      expect.assertions(2)
       const { close, localStoreManager, localStore, discoveryKey } = await open()
 
       // change something in the local store
-      localStore.dispatch({ type: 'ADD_TEACHER', payload: [teacher1, teacher2] })
+      localStore.dispatch({ type: 'ADD_TEACHER', payload: [teacher1] })
```

Comment:
We can restore the test with 2 teachers

Reply:
I'm going to be adding a bunch of tests, covering interactions with larger numbers of records etc.